docs/conf.py

```diff
@@ -213,3 +215,119 @@
 
 # If true, `todo` and `todoList` produce output, else they produce nothing.
 todo_include_todos = True
+
+# -- Options for autodoc extension --------------------------------------------
+autodoc_default_options = {
+    'inherited-members': True,
+}
+
+autodoc_member_order = 'groupwise'
+
+
+# Autodoc event handlers
+def autodoc_process_docstring(app, what, name, obj, options, lines):
+    """Function to handle the autodoc-process-docstring event.
+
+    Emitted when autodoc has read and processed a docstring. lines is a list of
+    strings – the lines of the processed docstring – that the event handler
+    can modify in place to change what Sphinx puts into the output.
+
+    More information on sphinx documentation:
+        http://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html
+        #event-autodoc-process-docstring
+
+    Args:
+        app: the Sphinx application object
+        what: the type of the object which the docstring belongs to (one of "module",
+            "class", "exception", "function", "method", "attribute")
+        name: the fully qualified name of the object
+        obj: the object itself
+        options: the options given to the directive: an object with attributes
+            inherited_members, undoc_members, show_inheritance and noindex that
+            are true if the flag option of same name was given to the auto directive
+        lines: the lines of the docstring
+    """
+
+    # Make class properties display matching the 'Args' format
+    if what == 'class':
+        make_properties_one_liner(lines)
+        make_fields_bolded(lines)
+
+    return lines
+
+
+def make_properties_one_liner(lines):
```

Comment:
As I said in my previous answers to your questions, I think we can remove this and just use better practices of not describing inputs in the bulleted list of properties.